lesson11/index.js

```diff
@@ -0,0 +1,68 @@
+const jsonUrl = 'https://myjson.dit.upm.es/api/bins/77w5';
+
+const ul = document.getElementById('js-list');
+
+const createLoading = () => {
+    const li = document.createElement("li");
+    const loadingImg = document.createElement('img');
+
+    li.style.listStyle = "none";
+    li.id = "js-loading";
+    loadingImg.src = 'loading-circle.gif';
+    ul.appendChild(li).appendChild(loadingImg);
+};
+
+const removeLoading = () => document.getElementById('js-loading').remove();
+
+const addList = (val) => {
+    const fragment = document.createDocumentFragment();
+    for (let i = 0; i < val.length; i++) {
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+    
+        a.href = `${val[i].a}.html`;
+        a.textContent = `${val[i].text}`;
+        img.src = val[i].img;
+        img.alt = val[i].alt;
+    
+        fragment.appendChild(li).appendChild(a).appendChild(img);
+    }
+
+    ul.appendChild(fragment);
+};
+
+/**
+ * Promiseを実行し、resolveの値を返す関数
+ * @return {promise object} 
+ */
+ const fetchData = (json) => {
+    return new Promise( resolve => {
+        setTimeout(() => {
+            resolve(json);
+        }, 3000);
+    });
+};
+
+/**
+ * Async関数
+ * responseで値が返ってきた後、addList関数にその値を渡す
+ */
+const execution = async (url) => {
```

Comment:
Thank you for review!
Changed functions name.
Since it is a function to create new data, I used "create".
[d653d7a](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/11/commits/d653d7a132516f29c8f34bab04f37b77c7c37ad5)